dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1901,6 +1901,29 @@ def test_diff_with_different_schema(self, *mocks):
 
         self.assertDictEqual(expected_diff, profile1.diff(profile2))
 
+    @mock.patch("dataprofiler.profilers.profile_builder.DataLabeler")
+    @mock.patch(
+        "dataprofiler.profilers.data_labeler_column_profile.DataLabelerColumn.update"
+    )
+    def test_diff_categorical_chi2_test(self, *mocks):
+        """Ensuring that chi2-test is bubbled up to the top
+        when running the top level profiler."""
+        df_categorical = pd.Series(["y", "y", "y", "y", "n", "n", "n"])
+        df_categorical2 = pd.Series(["y", "maybe", "y", "y", "n", "n", "maybe"])
+
+        profile1 = dp.StructuredProfiler(df_categorical)
+        profile2 = dp.StructuredProfiler(df_categorical2)
+
+        diff = profile1.diff(profile2)
+        expected_chi2_test_dict = {
+            "chi2-statistic": 2.342857142857143,
+            "df": 2,
+            "p-value": 0.3099238764710244,
+        }
+        self.assertDictEqual(
+            expected_chi2_test_dict, diff["data_stats"][0]["statistics"]["chi2-test"]
+        )
+
```

Comment:
new test to validate bubbling up `chi2-test` to the top level profile

Reply:
was chi2-test not covered before? I thought it existed as a metric before but now it's just being moved into `statistics`. So is there a test that needs to be removed as well?